tests/src/test/scala/ch/renku/acceptancetests/MinimalFunctionalitySpec.scala

```diff
@@ -22,6 +24,8 @@ class MinimalFunctionalitySpec
 
     implicit val loginType: LoginType = logIntoRenku
 
+    implicit val docsScreenshots = new DocsScreenshots(this, browser)
```

Comment:
I think this should be removed from here. See below on how to handle the functions that need this type as an argument.